Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1060 @@
+
+| 필드 | 타입 | 필수 | 설명 |
+| --- | --- | --- | --- |
+| like | boolean | ✅ | 좋아요를 취소합니다. 취소 시 게시물에서 좋아요가 1 줄어듭니다. |
+
+**Response (200 OK)**
+
+```json
+
+  "status": "success",
+  "data":{
+	    "title": "게시글 제목"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)